cmd/config/subcommand/config/console_flags.go

```diff
@@ -0,0 +1,20 @@
+package config
+
+import "github.com/flyteorg/flytestdlib/config"
+
+//go:generate pflags ConsoleConfig --default-var DefaultConsoleConfig --bind-default-var
+
+var (
+	DefaultConsoleConfig = &ConsoleConfig{}
```

Comment:
I've modified pyflyte run to consume console.endpoint from the flytectl coming file in https://github.com/flyteorg/flytekit/pull/1233 so that it doesn't need to special case localhost:30081. 

This section is required to validate and load the config file in flytectl even though the new section won't be consumed here.